parent/pom.xml

```diff
@@ -107,7 +107,7 @@
         <jsonPath.version>2.0.0</jsonPath.version>
         <commons-compress.version>1.4</commons-compress.version>
         <qpid.version>0.20</qpid.version>
-        <mongodb.version>2.11.4</mongodb.version>
+        <mongodb.version>3.0.3</mongodb.version>
```

Comment:
The `AbstractMongoDBServer` still uses version 2.6.5. Should this really be version 3.x of the client?